firefox-ios/Client/Frontend/Browser/Tabs/Views/RemoteTabsViewController.swift

```diff
@@ -489,4 +502,19 @@ class RemoteTabsTableViewController: UITableViewController,
             }
         }
     }
+
+    private func updateInsets() {
+        if shouldUseiPadSetup() {
+            (emptyView as? InsetUpdatable)?.updateInsets(top: view.safeAreaInsets.top, bottom: 0)
+        } else {
+            let emptyView = emptyView as? InsetUpdatable
+            let bottomInset = if (emptyView as? ExperimentRemoteTabsEmptyView) != nil {
+                DefaultTabTrayUtils().segmentedControlHeight + view.safeAreaInsets.bottom
+            } else {
+                DefaultTabTrayUtils().segmentedControlHeight
+            }
```

Comment:
We could access this from a `RemoteTabsEmptyViewProtocol` computed variable? 

Example:
```
protocol RemoteTabsEmptyViewProtocol: UIView, ThemeApplicable {
    var needsSafeArea: Bool { get }
```

Then we access it from `emptyView.needsSafeArea`

```
            let bottomInset = if emptyView.needsSafeArea {
                DefaultTabTrayUtils().segmentedControlHeight + view.safeAreaInsets.bottom
            } else {
                DefaultTabTrayUtils().segmentedControlHeight
            }
```